ui/components/app/asset-list/asset-list.js

```diff
@@ -98,19 +123,37 @@ const AssetList = ({ onClickAsset }) => {
       />
       {detectedTokens.length > 0 &&
         !istokenDetectionInactiveOnNonMainnetSupportedNetwork && (
-          <DetectedTokensLink setShowDetectedTokens={setShowDetectedTokens} />
+          <>
+            {process.env.MULTICHAIN ? (
+              <DetectedTokensBanner
+                actionButtonOnClick={() => setShowDetectedTokens(true)}
+                margin={4}
+              />
+            ) : (
+              <DetectedTokensLink
+                setShowDetectedTokens={setShowDetectedTokens}
+              />
+            )}
+          </>
         )}
       <Box marginTop={detectedTokens.length > 0 ? 0 : 4}>
-        <Box justifyContent={JustifyContent.center}>
-          <Typography
-            color={Color.textAlternative}
-            variant={TypographyVariant.H6}
-            fontWeight={FONT_WEIGHT.NORMAL}
-          >
-            {t('missingToken')}
-          </Typography>
-        </Box>
-        <ImportTokenLink />
+        {process.env.MULTICHAIN ? (
+          <MultichainImportTokenLink margin={4} />
+        ) : (
+          <>
+            <Box justifyContent={JustifyContent.center}>
+              <Text
+                color={Color.textAlternative}
+                variant={TextVariant.bodySm}
+                as="h6"
+              >
+                {t('missingToken')}
+              </Text>
+            </Box>
+
+            <ImportTokenLink />
+          </>
+        )}
       </Box>
       {showDetectedTokens && (
         <DetectedToken setShowDetectedTokens={setShowDetectedTokens} />
```

Comment:
Hmm it would be cool to create a story that works with the multichain env variable option so we can see both. If you have time I would suggest it. It would be very handy for @SaraCheikh to do a visual review of multichain work
https://storybook.js.org/docs/react/configure/environment-variables

Reply:
Is the thinking here that we'd see if other old CSS would mess up the display of the new work?  My concern here would be that it's extra work we'd possibly be removing when the flag is removed.  Am I missing something?

Reply:
I was hoping there would be a solution to turn on and off the environment variables to show the Multichain version of a component but I can't find anything so well have to resort to checking the local build in this case.